src/components/Video/Video.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+import Media from 'Media';
+
+type VideoProps = {
+  caption?: string;
+  credit?: string;
+  className?: string;
+  src: string;
+};
+
+export const Video = ({ caption, credit, className, src }: VideoProps) => {
+  const [loaded, setLoaded] = useState(false);
+
+  const onLoad = () => {
+    setLoaded(true);
+  };
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const onError = (e: any) => {
+    console.error('Video error', e);
+  };
```

Comment:
This isn't used anywhere - presumably it's there for use in the future?

Reply:
Good point - I've removed this and onLoad.

It may be useful in future but since video will be embedded in an iframe the error handling can be done at the YouTube end.